src/shared/services/dom-util.service.ts

```diff
@@ -32,6 +32,35 @@ export class DomUtilService {
       if (firstInput) {
         firstInput.focus();
         firstInput.select();
+      } else {
+        firstInput = el.querySelector('div.btn-group') as HTMLInputElement;
+        if (firstInput) {
+          // .open is used for opening the dropdownmenu
+          firstInput.classList.add('open');
+          firstInput = el.querySelector('div.btn-group input') as HTMLInputElement;
+          firstInput.focus();
```

Comment:
Amended the example, the main thing is to aggregate in the first part of the function the looking for the input, and at the end, if you found it, do the lines:

```javascript
firstInput.focus();
firstInput.select();
```